mycli/config.py

```diff
@@ -19,16 +19,40 @@ class CryptoError(Exception):
 
 logger = logging.getLogger(__name__)
 
-def load_config(usr_cfg, def_cfg=None):
-    cfg = ConfigObj()
-    cfg.merge(ConfigObj(def_cfg, interpolation=False))
-    cfg.merge(ConfigObj(expanduser(usr_cfg), interpolation=False))
-    cfg.filename = expanduser(usr_cfg)
+def read_config_file(f, base_config=None):
+    """Read and merge a config file.
 
-    return cfg
+    If the file is read successfully, the config object takes
+    on that filename.
+    """
+
+    config = ConfigObj() if base_config is None else base_config
+    try:
+        _config = ConfigObj(f, interpolation=False)
+        config.merge(_config)
+        if bool(_config) is True:
```

Comment:
Thank you for being explicit here. It is tempting to do `if bool(_config):` instead of `if bool(_config) is True:`, but I like that you've explicitly called this out. Makes it clear to read.